modal/cli/environment.py

```diff
@@ -38,10 +39,22 @@ def __rich__(self):
 @environment_cli.command(name="list", help="List all environments in the current workspace")
 def list(json: Optional[bool] = False):
     envs = environments.list_environments()
+
+    # determine which environment is currently active, prioritizing the local default
+    # over the server default
+    cfg_default = config.get("environment")
+    active_env = None
+    for env in envs:
+        if env.name == cfg_default:
+            active_env = env.name
```

Comment:
minor nit: if you initialize `active_env = config.get("environment")`, you can simplify this logic by getting rid of the first `if` statement